cfme/tests/configure/test_access_control.py

```diff
@@ -2979,3 +3025,59 @@ def test_select_edit_group(appliance, request):
         view.toolbar.configuration.open()
         assert view.toolbar.configuration.item_enabled(
             "Edit the selected Group"), '"Edit the selected Group" button is disabled'
+
+
+@pytest.mark.customer_scenario
+@test_requirements.rbac
+@pytest.mark.tier(2)
+@pytest.mark.meta(automates=[1730066])
+@pytest.mark.provider([EC2Provider], scope='function', selector=ONE_PER_TYPE)
+@pytest.mark.parametrize("tenant_custom_role", ["top", "child"], indirect=True)
+def test_aws_keypair_list(appliance, setup_provider, provider, tenant_custom_role, request):
+    """
+    Should able to view AWS keypair list as tenant_administrator
+    Bugzilla:
+        1730066
+    Polarion:
+        assignee: dgaikwad
+        casecomponent: Configuration
+        caseimportance: critical
+        initialEstimate: 1/4h
+        testSteps:
+            1. Login user admin user
+            2. add aws cloud provider
+            3. Copy the EvmRole_tenant_admin role to a new role (Since this role does not have the
+               Auth Key Pairs feature enabled)
+            4. In the new role, enable the Auth Key Pairs feature
+            5. Add new group to the newly created tenant admin role
+            6. Navigate to "Compute>Cloud>Key Pair" and add new "Key Pair" by using recently
+            added also set permisson
+            7. a) If the added groups belong to the Top-Level Tenant (Parent Tenant, Example
+                  "My Company"), navigate to "Compute>Cloud>Key Pair" page by login respective user.
+               b) if the added groups belong to one of the sub-tenant (Children tenant,
+                  example "My Company/new_tenant"), navigate to "Compute>Cloud>Key Pair" page by
+                  login respective user.
+        expectedResults:
+            1.
+            2.
+            3.
+            4.
+            5.
+            6.
+            7. a) should able see created Key Pair
+               b) should able see created Key Pair
+    """
+    user, group, _ = tenant_custom_role
+    keypair = appliance.collections.cloud_keypairs.create(
+        name=fauxfactory.gen_alphanumeric(),
+        provider=provider
+    )
+    assert keypair.exists
+    request.addfinalizer(keypair.delete)
+
+    view = navigate_to(appliance.collections.cloud_keypairs, 'All')
```

Comment:
optional: When we have `BaseEnity` object you wan access `CollectionEnity` with parent.
```
navigate_to(keypair.parent, 'All')
```

Reply:
Done